src/Manager/Environment/Install.tsx

```diff
@@ -14,34 +19,83 @@ import Button from './Button';
 import { install } from './effects/installEnvironment';
 import { isOnlyAvailable, version } from './environmentReducer';
 
-type Props = { environment: Environment };
+type Props = { environment: Environment; showExperimentalWarning: boolean };
 
-const Install = ({ environment }: Props) => {
+const Install = ({ environment, showExperimentalWarning }: Props) => {
     const dispatch = useDispatch();
     const anyInProgress = useSelector(isAnyToolchainInProgress);
+    const [showExperimentalDialog, setShowExperimentalDialog] = useState(false);
 
     const { platform } = process;
-    const onClick = (() => {
+    const onClick = () => {
         switch (platform) {
             case 'darwin':
-                return () => dispatch(install(environment, false));
+                return dispatch(install(environment, false));
             case 'linux':
-                return () =>
-                    dispatch(showConfirmInstallDirDialog(version(environment)));
+                return dispatch(
+                    showConfirmInstallDirDialog(version(environment))
+                );
             case 'win32':
-                return () =>
-                    dispatch(showConfirmInstallDirDialog(version(environment)));
+                return dispatch(
+                    showConfirmInstallDirDialog(version(environment))
+                );
         }
-    })();
+    };
 
     return !isOnlyAvailable(environment) ? null : (
-        <Button
-            icon="x-mdi-briefcase-download-outline"
-            onClick={onClick}
-            label="Install"
-            variant="secondary"
-            disabled={anyInProgress}
-        />
+        <>
+            <GenericDialog
+                isVisible={showExperimentalDialog}
+                headerIcon="alert"
+                title="Experimental Toolchain and SDK"
```

Comment:
I don't think the toolchain and SDK are experimental, but the support for them, no? Also, wasn't only the support for the SDK experimental?

Reply:
```suggestion
                title="Experimental Toolchain and SDK for VSCode"
```

Reply:
Your suggestion does not answer my question :)

Reply:
The SDK/Toolchain are experimental when in use with VSCode

Reply:
@greg-fer  ok for the above

Reply:
@kylebonnici :
```
Support for Toolchain and SDK experimental in VS Code
```